Update linter and fix issues found

albertvaka opened the pull request:
Bumps the linter docker image and fixes the following warnings:
- File permissions not being explicit.
- `role_name` and `namespace` missing from metadata.
- Wildcards used in pre/post tasks are not expanded (confirmed manually in 2.6 and 2.10)-> I removed the tasks since they never worked so nobody must be using them.

albertvaka committed:
Update lint job

albertvaka committed:
Fix file permissions

albertvaka committed:
Add role_name and namespace to galaxy_info

albertvaka committed:
Remove pre/post tasks

It never (testesd in 2.6 and 2.10) worked: wildcards aren't expanded

KSerrania reviewed (commented):
LGTM.

About the file permissions: we're already making the Agent 6/7 main config file not world-readable. Should we do the same for the integrations config files, the config dir itself? And should we do the same for the Agent 5 config too?

albertvaka commented on tasks/agent5-linux.yml:
```diff
@@ -67,12 +69,6 @@
     dest: "/etc/dd-agent/conf.d/{{ item }}.yaml"
     owner: "{{ datadog_user }}"
     group: "{{ datadog_group }}"
+    mode: 0644
```
```suggestion
    mode: 0644 #FIXME: should have permissions set to only be readable by owner
```

albertvaka commented on tasks/agent5-linux.yml:
```diff
@@ -3,13 +3,15 @@
     dest: /etc/dd-agent/datadog.conf
     owner: "{{ datadog_user }}"
     group: "{{ datadog_group }}"
+    mode: 0644
```
```suggestion
    mode: 0644 #FIXME: should have permissions set to only be readable by owner
```

albertvaka committed:
Apply suggestions from code review

albertvaka commented:
I've just added a FIXME for A5, the same way I added a FIXME for Windows in the original changes. Those are breaking changes so they will have to wait until a new major version. The Windows one is a bit more worrying than A5. About the directories themselves, I don't think it's a security issue that they are accessible by anyone.

kayayarai reviewed (commented):
No docs review required
